.gitignore

```diff
@@ -58,5 +57,4 @@ yarn.lock
 package-lock.json
 pnpm-lock.yaml
 camille-bot/.dev.vars
-camille-bot/wrangler.toml
```

Comment:
This also doesn't need to be ignored. The KV values in here can only be accessed inside your organization/team/user account. The only sensitive data in here would be the [[vars]] but we move those into `.dev.vars` which is ignored.